pkg/importer/dataStream.go

```diff
@@ -412,7 +413,7 @@ func (d *dataStream) isoSize() (int64, error) {
 	const (
 		vdTypeOff       = 0
 		typeLen         = 1
-		vdIdOff         = 1
+		vdIDOff         = 1
```

Comment:
It's personal choice until you run golint against it.  Honestly most of these aren't just things I've pulled out of the air, they're actual standardized *things*.  This isn't my hill to die on, I just volunteered to turn on golint; if we don't want it that's ok too but I do think it's valuable and very much worth while even if it is annoying at first.  I'm a very firm believer in gofmt, golint and govet, ESPECIALLY in an Open Source project where you hope to have a community of developers.

Reply:
Agree. Take my review comments with a grain of salt because I didn't realize, while reviewing your prs last Fri, that the changes you made were necessary for `golint` to run.